layouts/page/guideline.html

```diff
@@ -0,0 +1,204 @@
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!--list of departments-->
+    <section class="bg-body-secondary mb-5">
+        <div class="container mt-2 py-5">
+            <!-- Executive Director row -->
+            <div class="row justify-content-center">
+                {{ $depFirstRow := slice "Executive Director" "Secretarial Director" }}
+                {{ range $depFirstRow }}
+                    <div class="col-12 col-md-4 mb-3">
+                        <div style="border-radius: 15px;" class="col bg-secondary py-3 px-3 d-flex align-items-center justify-content-center">
+                            <p class="text-primary" style="font-weight: 600;">
+                                {{ . }}
+                            </p>
+                        </div>
+                    </div>
+                {{ end }}
+            </div>
+
+            <!-- Department rows -->
+            <div class="row justify-content-center g-3">
+                {{ range $index, $element := $guideline }}
+                    {{ if gt $index 1 }}
+                        <div class="col-md-4">
+                            <div style="border-radius: 15px;" class="col bg-secondary py-3 px-3 d-flex align-items-center justify-content-center">
+                                <p class="text-center mb-0 text-primary" style="font-weight: 600;">
+                                    {{ .department }}
+                                </p>
+                            </div>
+                        </div>
+                    {{ end }}
+                {{ end }}
+            </div>
+        </div>
+    </section>
+
+    <!--info of each departments-->
+    <section class="container mb-5">
+        <div class="py-5 row g-3 justify-content-center">
+            {{ range $guideline }}
+                <div class="col-md-6">
+                    <div class="bg-body-secondary mb-2 border-0 px-3 py-3" style="border-radius: 15px">
+                        <!--image-->
+                        <div class="d-flex justify-content-center mb-3">
+                            <img src="{{ absURL .icon }}" alt="{{ .department }} Icon" class="img-fluid"/>
+                        </div>
+
+                        <!--dropdown button-->
+                        <div class="dropdown d-block text-center">
+                            <button 
+                                style="border-radius: 10px; width: 100%"
+                                class="dropdown-toggle py-2 px-3 text-primary border-0 fw-bold text-wrap"
+                                type="button"
+                                data-bs-toggle="dropdown"
+                                data-bs-display="static"
+                                data-bs-auto-close="outside"
+                                aria-expanded="false"
+                            >
+                                {{ .department }}
```

Comment:
```suggestion
                                {{ .department -}}&nbsp;
```
the dash removes trailing spaces, &nbsp; prevent line breaking

![image](https://github.com/user-attachments/assets/313833a9-a863-4c5a-ae62-155f0158ae7c)

![image](https://github.com/user-attachments/assets/7ca00358-3cfb-4849-bde5-150c7754e1c8)